### ISSUE TITLE
fabri8:apply doesn't create multiple routes if multiple services are created

### ISSUE THREAD

**Issue by rawlingsj** [fabric8io/fabric8#4633]
The taiga app defines multiple services but when using fabric8:apply it will only create an openshift route for one of the new services.

**Comment by jimmidyson** [comment]
Do you actually need routes for every service?

**Comment by rawlingsj** [comment]
Actually no, but the service it chose to create a route for was not the one I wanted.  

On google cloud the service type `LoadBalancer` is used when we want an external loadbalancer created, maybe we could use the same  when we want to create a route?

**Comment by jstrachan** [comment]
Good idea! I wonder if the LoadBalancer should be the indicator to create a route (then an external IP is also created on GKE). 

The other option is a custom annotation on services requiring a route - though I like the former if it doesn't cause any issues

**Comment by jstrachan** [comment]
Relates to this issue
https://github.com/openshift/origin/issues/2026
Am hoping eventually OpenShift natively creates routes for us

**Comment by rawlingsj** [comment]
Ah yeah that would definitely be better if openshift could create the routes.
